packages/nextjs/app/builders/0xA35D2C518710D3f953Ce4F69CDDEAaFc0c6b156c/page.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+
+import React, { useState } from "react";
+import { Check, Copy, ExternalLink } from "lucide-react";
+
+const Web3Profile = () => {
```

Comment:
Since there will be many profiles, give your page a unique name. Use your name or nickname.

And it's also a good idea to explicitly add the `NextPage` type.